README.md

```diff
@@ -140,27 +178,66 @@ _After_ you have ***contributed*** to the project,
 _e.g: by commenting on or creating an issue_, <br />
 -->
 If you've found the project interesting, helpful or useful in _any way_,
-_please_ share the love by starring it! <br />
+_please_ ***star*** the repository on GitHub! <br />
 Stars show _other_ members in the developer community
 that it's a _**worthwhile project** or **learning resource**_
 and one that can offer value to other developers like you! 🌟
 
 
 <br />
 
-## Questions?
+# Questions?
```

Comment:
Should all of this come *before* the rest of the content? Or referred to here for a link to the bottom of the page?